documentation/index.md

```diff
@@ -1,11 +1,9 @@
-<center>![logo](./media/design/logo.png)</center>
+<span class="centered">![BetonQuest Logo](./media/design/logo.png)</span>
```

Comment:
In the github preview, this is not centered. Is it in mkdocs?

Aslo: can we use this url in the README.md?

Reply:
Yes it is.

We can but I have no idea how to use my CSS in the README.